walkthroughs/week-5-tdd/project/src/main/java/com/google/sps/FindMeetingQuery.java

```diff
@@ -76,7 +76,7 @@ public int compare(Map.Entry<String, Integer> o1, Map.Entry<String, Integer> o2)
         ArrayList<Integer> optionalAttendeesOptimalAvailableTimes = new ArrayList<Integer>();
         int count = 0;
         for (Map.Entry<String, Integer> entry : optionalAttendeesAvailableTimeValuesSorted.entrySet()) {
-            switch (entry.getKey().charAt(1)) {
+            switch (entry.getKey().charAt(0)) {
```

Comment:
definitely more robust. however, do you even use the value of i from the loop on line 59? maybe you don't need it at all?

also I can't seem to respond to your question, but i think "start" and "end" would be a good idea, if you can remove the i entirely (so this would just be entry.getKey()). if not, then i think it's fine as-is.

Reply:
Because I used the value of i to make the key unique, I think that I will still need it if I use “S” and “E” to store the start or end information.

Reply:
Ohh, I see. That clears it up. S and E sgtm!